packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/flex.dart

```diff
@@ -340,70 +367,105 @@ class _StoryOfYourFlexWidgetState extends State<StoryOfYourFlexWidget> {
+        child: DropdownButton(
+          value: selected,
+          isExpanded: true,
+          itemHeight: axis == Axis.vertical ? 100.0 : null,
+          selectedItemBuilder: (context) {
+            return [
+              for (var alignment in alignmentEnumEntries)
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.end,
+                  crossAxisAlignment: CrossAxisAlignment.center,
+                  textDirection: axis == Axis.horizontal
+                      ? TextDirection.ltr
+                      : TextDirection.rtl,
+                  children: <Widget>[
+                    Expanded(
+                      child: Container(
+                        child: Text(
+                          describeEnum(alignment),
+                          style: TextStyle(color: color),
+                          textAlign: TextAlign.center,
                         ),
-                        Image.asset(
+                      ),
+                    ),
+                    Expanded(
+                      child: RotatedBox(
+                        quarterTurns: axis == Axis.vertical ? 2 : 0,
+                        child: Image.asset(
                           (axis == direction)
                               ? mainAxisAssetImageUrl(alignment)
                               : crossAxisAssetImageUrl(alignment),
                           height: axisAlignmentAssetImageHeight,
                           fit: BoxFit.contain,
                         ),
-                      ],
+                      ),
                     ),
-                  )
-              ],
-              onChanged: (Object newSelection) async {
-                if (axis == direction) {
-                  properties.mainAxisAlignment = newSelection;
-                } else {
-                  properties.crossAxisAlignment = newSelection;
-                }
-                final service =
-                    await properties.node.diagnostic.inspectorService;
-                final arg = properties.node.diagnostic.valueRef;
-                await service.invokeTweakFlexProperties(
-                  arg,
-                  properties.mainAxisAlignment,
-                  properties.crossAxisAlignment,
-                );
-                setState(() {});
-              },
-            ),
-          )
-        ],
+                  ],
+                )
+            ];
+          },
+          items: [
+            for (var alignment in alignmentEnumEntries)
+              DropdownMenuItem(
+                value: alignment,
+                child: Container(
+                  padding: const EdgeInsets.symmetric(vertical: margin),
+                  child: Row(
+                    mainAxisAlignment: MainAxisAlignment.end,
+                    crossAxisAlignment: CrossAxisAlignment.center,
+                    textDirection: axis == Axis.vertical
+                        ? TextDirection.rtl
```

Comment:
nit: probably don't change the textDirection. Change the alignment instead.
TextDirection should be about I18N not tweaking where to place text in a graphical UI.